examples/morpheusvm/tests/load/load_test.go

```diff
@@ -458,17 +467,33 @@ var _ = ginkgo.Describe("load tests vm", func() {
 
 		ginkgo.By("producing blks", func() {
 			start := time.Now()
+			acceptCalls := []func(){}
 			for {
-				blk := produceBlock(instances[0])
+				blk, accept := produceBlock(instances[0])
 				if blk == nil {
 					break
 				}
+				acceptCalls = append(acceptCalls, accept)
```

Comment:
How big can this get?

Reply:
This depends on the number of transactions provided. In our existing tests, it is ~50.